nerd-icons-completion.el

```diff
@@ -139,8 +140,12 @@ PROP is the property which is looked up."
   "Add icons to completion candidates."
   :global t
   (if nerd-icons-completion-mode
-      (advice-add #'completion-metadata-get :around #'nerd-icons-completion-completion-metadata-get)
-    (advice-remove #'completion-metadata-get #'nerd-icons-completion-completion-metadata-get)))
+      (progn
+	(advice-add #'completion-metadata-get :around #'nerd-icons-completion-completion-metadata-get)
+	(advice-add (compat-function completion-metadata-get) :around #'nerd-icons-completion-completion-metadata-get))
```

Comment:
Use spaces instead of tabs for consistency.

And is it necessary to add advice twice? In my case, adding advice to `(compat-function completion-metadata-get)` is just fine.

Reply:
I've replaced the unnoticed tabs to spaces, thanks. 
Concerning the 'double advice', both functions must be advised since they are not necessarily the same. As pointed out by @minad, they only coincide on Emacs 30 and newer.

Reply:
Yes, I think “double advice” is necessary, so it works even when `vertico-mode` is not enabled.